assets/docs/fragments/cli-installation.md

```diff
@@ -5,12 +5,12 @@ Follow the [AsyncAPI CLI installation](https://github.com/asyncapi/cli#installat
 <summary>MacOS</summary>
 
 `brew`
-<br/>
-You can install in MacOS via brew: `brew install asyncapi`.
+
+> You can install in MacOS via brew: `brew install asyncapi`.
```

Comment:
@nikhilkalburgi can by any means this command be bold here

Reply:
Hi @sambhavgupta0705 ,
Yes, it is possible. 

This is how it will look like:
![image](https://github.com/asyncapi/website/assets/70331875/7036305e-f9dc-4a8a-a344-366c48423ab1)
I have updated my PR.

Reply:
Works